TRIAGE_GUIDE.md

```diff
@@ -0,0 +1,90 @@
+* The issue is an actual issue, rather than a request for help or an issue for another SIG.
+
+
+# Process
+SIG-Network will triage issues once a week on [Thursdays](https://lists.o3de.org/g/o3de-calendar/viewevent?repeatid=39342&eventid=1263668&calstart=2022-01-20). Anyone is welcome to attend. Triage will be led by SIG chair or maintainer.
+
+Triaging aims to:
+* Ensure issues in backlog are in a ready state for the community to take action upon.
+* Ensure load is balanced across SIG maintainers and participants.
+* Involve the SIG-Network community so all can participate.
+
+If time permits, on the day of triage and before the meeting, create a new thread in SIG-Network and add triage links below.
+* Set the thread to automatically archive after 24 hours.
+
+## Triage Links
+* Main O3DE repository: https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage+label%3Asig%2Fnetwork 
+* Multiplayer Sample: https://github.com/o3de/o3de-multiplayersample/issues
+* NetSoak Test: https://github.com/o3de/o3de-netsoaktest/issues
+
+## Brief Overview
+1. Join the SIG-Network discord voice channel
+2. Start with main repro link
+3. Process all new main repository issues. New issues in main repository should have labels `needs-triage` and `sig/network`
+   1. Announce issue number and title to those in Discord voice channel so others can follow along
+4. Ensure issue is for SIG-Network. If the issue is not for SIG-Network, remove the `sig/network` label and comment on the issue. If the correct SIG is known 
+    assign it to that SIG otherwise add the `needs-sig` label so the general O3DE issue triage meeting can triage and find the appropriate owners.
+5. Review the issue and comments and see if it can be accepted
+6. Review the technical implications. If a large change, issue should become an RFC, ask requestor to bring issue back as RFC or to make a feature request, if that would be more appropriate.
+7. Assign a reviewer, if required, to handle follow-up comments, to reproduce the issue or ask questions.
+8. If issue is rejected, assign commenter to reject issue. 
+9. If issue is accepted, remove `needs-triage` label, set priority for issue and add `triage/accepted` label.
+10. For Multiplayer and NetSoak issues, look for issues less than 14 days old that do not have priorities attached.
+    1. These repositories do not have the full set of labels so some of the process above may not apply directly.
+
+
+If time permits:
+* Review any open [blocker](https://github.com/o3de/o3de/issues?q=is%3Aissue+is%3Aopen+label%3Asig%2Fnetwork+label%3Apriority%2Fblocker) and [critical](is:issue is:open label:sig/network label:priority/critical) issues in the main repository:
+  * Ensure priority is still valid
+  * Assign any required commentators or ask for updates
+* Review any issue open for more than [90 days](is:issue is:open label:sig/network sort:created-asc) and see if its still valid.
```

Comment:
The 90 days link won't work, it needs the full URL

Reply:
Good catch, copy-n-pasta fail.